packages/popover/src/Popover.story.tsx

```diff
@@ -40,15 +76,24 @@ export default {
     children: {
       control: false,
     },
+    refButtonPosition: {
+      options: ['centered', 'top', 'right', 'bottom', 'left'],
+      control: { type: 'select' },
+    },
```

Comment:
I just noticed that this arg is only being used in the scrollableContainer story, so it would be better for this arg to be defined just on that story, not here on the story level.

Reply:
This should be used for both stories, i'll updated the other story